client/my-sites/stats/stats-module/style.scss

```diff
@@ -1,3 +1,8 @@
+@import "@wordpress/base-styles/breakpoints";
+
+$grid-horizontal-gutters: 24px;
```

Comment:
Those variables could be shared between the grids if we used a mixin with a default parameter value.